draft-irtf-cfrg-frost.md

```diff
@@ -547,17 +547,17 @@ each Signer then runs the following procedure to produce its own signature share
   - nonce_i, pair of Scalar values (d, e) generated in round one.
   - comm_i, pair of Element values (D, E) generated in round one.
   - msg, the message to be signed (sent by the Coordinator).
-  - commitment_list = [(j, x_j, y_j), ...], a list of commitments issued by each signer,
+  - commitment_list = [(j, D_j, E_j), ...], a list of commitments issued by each signer,
     where each element in the list indicates the signer index j and their
-    two commitment Element values (x_j, y_j). This list MUST be sorted in ascending order
+    two commitment Element values (D_j, E_j). This list MUST be sorted in ascending order
```

Comment:
I think your branch is behind, we renamed these

Reply:
I just checked again, and it's `(x_j, y_j)` in `master`, and my branch is `1` commit ahead of `cfrg:master`. Let me know if I'm missing something, thanks.

Reply:
We're referring to these as `hiding_nonce_commitment` and `binding_nonce_commitment` -- I'd prefer we use those names.

Reply:
OK, I will change to those names